CMakeLists.txt

```diff
@@ -232,6 +232,26 @@ if(WIN32)
     endif(CMAKE_SIZEOF_VOID_P EQUAL 8)
 endif(WIN32)
 
+# Setup FMA Capabilities Flag
+IF(UNIX)
+    MESSAGE(STATUS "Checking for FMA support...")
+    IF(NOT APPLE)
```

Comment:
why not swap the ordering here and do `IF(APPLE)` ... really just a small tweak, but IMHO makes it a little more readable.